test/rpc_test.py

```diff
@@ -227,8 +227,9 @@ def test_self_add(self):
 
     @mock.patch.object(torch.distributed.autograd, "_init")
     @mock.patch.object(torch.distributed.rpc.api, "_init_rref_context")
+    @mock.patch.object(torch.distributed.rpc.api, "_init_rpc_agent")
     def test_register_rpc_backend_and_init_rpc_backend(
-        self, mock_init_rref_context, mock_dist_autograd_init
+        self, mock_init_rref_context, mock_dist_autograd_init, mock_rpc_agent
```

Comment:
this is wrong. the inner decorator injects the first argument, the outer decorator injects the last argument